### DISCUSSION TITLE
markdown tables in jupyter notebook markdown cell cannot be normally displayed

### DISCUSSION THREAD

**Discussion by krim0217** [quarto-dev/quarto-cli#11516] [Q&A]
### Bug description

enable the quarto extension in VSCode.
![Image](https://github.com/user-attachments/assets/05dc8777-89e2-4c63-b589-55b1f54da684)
open a notebook and write a table in a markdown cell.
![Image](https://github.com/user-attachments/assets/ba743624-db3c-4fc1-ae8a-e47f19bb73f8)
run the cell.
![Image](https://github.com/user-attachments/assets/3c3e47bf-82d8-4e3c-bb98-37180d20e5ca)
the table has disappeared.

disable the quarto extension.
![Image](https://github.com/user-attachments/assets/2b208171-b0ec-4e3d-baf0-a68b2cc001bc)
re-open the notebook.
![Image](https://github.com/user-attachments/assets/4f0e17fd-47ff-4b0d-8765-180adc5e4935)
the table is normally displayed.

### Your environment

-IDE: Visual Studio Code 1.95.3
-OS: Windows 10

### Quarto check output

Quarto 1.5.57
[>] Checking versions of quarto binary dependencies...
      Pandoc version 3.2.0: OK
      Dart Sass version 1.70.0: OK
      Deno version 1.41.0: OK
      Typst version 0.11.0: OK
[>] Checking versions of quarto dependencies......OK
[>] Checking Quarto installation......OK
      Version: 1.5.57
      Path: D:\Code\Quarto\bin
      CodePage: 936

[>] Checking tools....................OK
      TinyTeX: (not installed)
      Chromium: (not installed)

[>] Checking LaTeX....................OK
      Using: Installation From Path
      Path: D:\texlive\2024\bin\windows
      Version: 2024

[>] Checking basic markdown render....OK

[>] Checking Python 3 installation....OK
      Version: 3.12.3 (Conda)
      Path: D:/Code/anaconda3/python.exe
      Jupyter: 5.7.2
      Kernels: ir, python3

[>] Checking Jupyter engine render....OK

[>] Checking R installation...........OK
      Version: 4.4.1
      Path: D:/Code/R/R-4.4.1
      LibPaths:
        - D:/Code/R/R-4.4.1/library
      knitr: 1.48
      rmarkdown: 2.29

[>] Checking Knitr engine render......OK

**Comment by mcanouil** [comment]
Thanks.

Please take a look at the VScode extension for already opened issues and if none, open one there.

https://github.com/quarto-dev/quarto


This repository is for Quarto CLI.
